main.py

```diff
@@ -121,21 +126,33 @@ def main():
                 logging.info('Song added')
                 logging.debug(playlist_add)
 
-            # thrown when 'tracks' is missing    
+            # thrown when 'tracks' is missing
             except AttributeError:
                 logging.info("This track does not have metadata - probably uploaded. Skipping")
-                with open('./errored-tracks.log', 'a') as file:
-                    file.writelines(f'playist: {name}\n')
+                try:
+                    with open('./errored-tracks.log', 'a') as file:
+                        file.writelines(f'playlist: {name}\n\n')
+                except:
+                    logging.info("Error writing playlist to errored list.  It most likely has special characters.")
```

Comment:
Noticed that logging would crash the entire script if either the playlist or track had special characters (e.g. korean, japanese, russian, etc.).